src/doc/rustdoc/src/advanced-features.md

```diff
@@ -46,3 +46,6 @@ pub struct BigX;
 
 Then, when looking for it through the `rustdoc` search, if you enter "x" or
 "big", search will show the `BigX` struct first.
+
+There are some limitations on the doc alias names though: you can't use `"` or whitespaces or
+tabs or backlines.
```

Comment:
I think by 'backlines' you meant 'newlines', but both that and spaces are whitespace.
```suggestion
There are some limitations on the doc alias names though: you can't use `"` or whitespace.
```

Reply:
Oh good point.